visualization/dashboard/src/MLIDashboard/Controls/Scatter/ScatterUtils.ts

```diff
@@ -354,6 +354,36 @@ export class ScatterUtils {
         return foundOption ? foundOption.key.toString() : undefined;
     }
 
+    public static updatePropsForSelections(plotlyProps: IPlotlyProperty, selectedRow: number): IPlotlyProperty {
+        if (selectedRow === undefined) {
+            plotlyProps.data.forEach(trace => {
+                _.set(trace, 'marker.line.width', [0]);
+                _.set(trace, 'selectedpoints', null); 
+            });
+            return _.cloneDeep(plotlyProps);
+        }
+        const selection = selectedRow !== undefined ? selectedRow.toString() : undefined;
+        plotlyProps.data.forEach(trace => {
+            let selectedIndexes: number[] = [];
+            let newWidths: number[] = [0];
+            if ((trace as any).customdata) {
+                const customData = ((trace as any).customdata as string[]);
```

Comment:
(trace as any).customdata thing, also I see a lot of repetition in the code here, might be good to consider moving some to common functions?

Reply:
this is v1 dashboard, I duplicated a function to unblock the merge. Not worth refactoring, its a throwaway function